src/composables/use-match-routes.ts

```diff
@@ -1,36 +1,34 @@
-import {
-  useContext,
-  ref,
-  useRoute,
-  useRouter,
-  Ref,
-} from "@nuxtjs/composition-api"
+import { ref, useRoute, useRouter, Ref } from "@nuxtjs/composition-api"
 
 import { ALL_MEDIA, searchTypes, supportedSearchTypes } from "~/constants/media"
 import usePages from "~/composables/use-pages"
 
 /**
  * Reactive property that returns true only on the matching routes.
- * Note that routes are matched by name, not the url path.
- *
- * Routes are also localized before comparison, so 'search' becomes
- * 'search__en', for example.
+ * Note that routes are matched by the url path.
```

Comment:
This comment might be out of sync. The code below suggests that they are still matched by a substring of the route's name, not the URL.

Reply:
Great catch, thank you!